interface.html

```diff
@@ -44,7 +44,7 @@
       <div id="screen-form">
         <div id="screen-list" class="form-group">
           <div class="col-sm-4 control-label">
-            <label for="page">Select a screen</label>
+            <label for="page">Select a screen {{#if options.required}}(Required){{/if}}</label>
```

Comment:
Let's use a parameter specific to page selection. This way we can have other required fields in the future.

```suggestion
            <label for="page">Select a screen {{#if options.pageRequired}}(Required){{/if}}</label>
```